src/symbol/glyph_source.js

```diff
@@ -5,6 +5,9 @@ const verticalizePunctuation = require('../util/verticalize_punctuation');
 const Glyphs = require('../util/glyphs');
 const GlyphAtlas = require('../symbol/glyph_atlas');
 const Protobuf = require('pbf');
+const TinySDF = require('@mapbox/tiny-sdf');
+const isChar = require('../util/is_char_in_unicode_block');
+/* eslint-disable new-cap */
```

Comment:
Some recent change should make this unnecessary? It's still used in `script_detection`, and using the .eslintrc that's currently checked in, I still get warnings using `is_char_in_unicode_block`.